test/e2e/storage/csi_volumes.go

```diff
@@ -81,7 +81,8 @@ var _ = Describe("E2E", func() {
 				// We use 16Mi size volumes because this is the minimum size supported
 				// by xfs filesystem's allocation group
 				// Ref: http://man7.org/linux/man-pages/man8/mkfs.xfs.8.html
-				claimSize: "16Mi",
+				claimSize: "110Mi",
```

Comment:
The above comment is not valid anymore. And the filesizes are depend on `testsuites.DriverInfo.MaxFileSize`, currently we are setting it to `testpatterns.FileSizeMedium` which is 100M. I believe It would be good if `claimSzie` is calculated from that, say MaxFileSize + SomeBuffer.

Reply:
Kubernetes 1.17 will dynamically calculate the volume size based on constraints of the test and of the driver.

I agree that the comment needs to be updated here.

Reply:
I changed comment to mention 110MB, retaining also most of reasoning for 16 MB (is that needed now or can be deleted?).
About using base value: if base is 100 MB and we need to add some hard-coded const to it in the code, then the result is not much better than using hard coded end value of 100MB, right? 
We are just unlucky here that our case needs slightly more than base value of 100 MB.